klayout/drc/testing/testcases/unit/drc_bjt.svg

```diff
@@ -0,0 +1,90 @@
+<polygon id="0x55eeb5d1fbb0" class="l12d0" points="-502.42,51.9 -502.42,177.8 -281.22,177.8 -281.22,51.9"/>
+<polygon id="0x55eeb5e698a0" class="l12d0" points="-1300.82,294.1 -1300.82,420 -1079.62,420 -1079.62,294.1"/>
+<polygon id="0x55eeb5e69910" class="l12d0" points="-502.42,294.1 -502.42,420 -281.22,420 -281.22,294.1"/>
+<polygon id="0x55eeb5edff30" class="l2d222" points="-1334.84,264.93 -1334.84,446.12 -992.4,446.12 -992.4,264.93"/>
+<polygon id="0x55eeb5eaf380" class="l127d5" points="-1282.22,70.5 -1282.22,160.9 -1076.57,160.9 -1076.57,70.5"/>
+<polygon id="0x55eeb5fb95c0" class="l127d5" points="-889.26,70.5 -889.26,160.9 -683.61,160.9 -683.61,70.5"/>
+<polygon id="0x55eeb5fb9630" class="l127d5" points="-483.82,70.5 -483.82,160.9 -278.17,160.9 -278.17,70.5"/>
+<polygon id="0x55eeb60123a0" class="l127d5" points="-1300.82,294.1 -1300.82,420.1 -1076.57,420.1 -1076.57,294.1"/>
+<polygon id="0x55eeb6012410" class="l127d5" points="-889.36,312.7 -889.36,403.1 -683.61,403.1 -683.61,312.7"/>
+<polygon id="0x55eeb60fc3d0" class="l127d5" points="-483.82,312.7 -483.82,403.1 -278.27,403.1 -278.27,312.7"/>
+<polygon id="0x55eeb60fc440" class="l58d0" points="-296.49,150.05 -296.49,151.25 -301.29,151.25 -301.29,153.65 -296.49,153.65 -296.49,154.85 -294.09,152.45"/>
+<polygon id="0x55eeb60fc4b0" class="l58d0" points="-290.74,150.05 -293.14,152.45 -290.74,154.85 -290.74,153.65 -285.94,153.65 -285.94,151.25 -290.74,151.25"/>
+<polygon id="0x55eeb60cbd10" class="l58d0" points="-85.04,137.45 -87.44,139.85 -85.04,142.25 -85.04,141.05 -80.24,141.05 -80.24,138.65 -85.04,138.65"/>
+<polygon id="0x55eeb5f603f0" class="l58d0" points="-478,137.45 -480.4,139.85 -478,142.25 -478,141.05 -473.2,141.05 -473.2,138.65 -478,138.65"/>
+<polygon id="0x55eeb5f604f0" class="l58d0" points="-683.7,150.05 -686.1,152.45 -683.7,154.85 -683.7,153.65 -678.9,153.65 -678.9,151.25 -683.7,151.25"/>
+<polygon id="0x55eeb5d61f40" class="l58d0" points="-689.45,150.05 -689.45,151.25 -694.25,151.25 -694.25,153.65 -689.45,153.65 -689.45,154.85 -687.05,152.45"/>
+<polygon id="0x55eeb60c04e0" class="l58d0" points="-1094.89,150.05 -1094.89,151.25 -1099.69,151.25 -1099.69,153.65 -1094.89,153.65 -1094.89,154.85 -1092.49,152.45"/>
+<polygon id="0x55eeb60c05e0" class="l58d0" points="-1089.14,150.05 -1091.54,152.45 -1089.14,154.85 -1089.14,153.65 -1084.34,153.65 -1084.34,151.25 -1089.14,151.25"/>
+<polygon id="0x55eeb5d05fd0" class="l58d0" points="-883.44,137.45 -885.84,139.85 -883.44,142.25 -883.44,141.05 -878.64,141.05 -878.64,138.65 -883.44,138.65"/>
+<polygon id="0x55eeb5e7ef30" class="l2d222" points="-526.14,264.93 -526.14,446.12 -183.7,446.12 -183.7,264.93"/>
+<polygon id="0x55eeb5edfec0" class="l2d222" points="-941.54,264.93 -941.54,446.12 -599.1,446.12 -599.1,264.93"/>
+<text id="0x55eeb5e6c160" class="l11t222" text-anchor="start" dominant-baseline="text-after-edge" transform="translate(-1304.9 278.48) scale(1 -1)">BJT.1</text>
```

Comment:
looks like the text is backward is that expected?

Reply:
@proppy We generate those from GDS. I don't believe we have control over how the text is done.

Reply:
is the text also backward in the gds?

Reply:
@proppy You mean it's mirrored? 

BTW, I don't think that matters if it's mirrored or not. It doesn't have any effect on the functionality or testing.

@FaragElsayed2 Could you check if `BJT.1` text is mirrored?

Reply:
related #56